### ISSUE TITLE
need to keep the instance in stop state and also stop the instance at 7 PM

### ISSUE THREAD

**Issue by lovepurohit** [awslabs/aws-instance-scheduler#148]
Hi,

I am trying to configure a schedule that keeps the instance in stopped state and is someone manually starts the instance, it should stop the instance at 7 PM. 

I have configured the schedule for the above task like this:
{
  "description": {
    "S": "stop hours"
  },
  "name": {
    "S": "stop"
  },
  "override_status": {
    "S": "stopped"
  },
  "periods": {
    "SS": [
      "office-hours"
    ]
  },
  "timezone": {
    "S": "US/Eastern"
  },
  "type": ""schedule""
}

office hours contain the period to stop the instance at 7 PM daily including the weekend.

Can you please help me and let me know if the above schedule is correct and will work as per the requirements?

Thank you

**Comment by ericquinones** [comment]
Hi @lovepurohit 
Did you change the `office-hours` period that was configured automatically when the solution was launched? Also, have you tested creating a period with an `endtime` only that was set to 7PM?

Please review [this section](https://docs.aws.amazon.com/solutions/latest/instance-scheduler/components.html#period-rules) of our deployment guide as I think this is the behavior you need.

Thank you

**Comment by mm534** [comment]
Hi, having a similar issue. The docs seem to suggest this should work with override_status set to stopped. For the scenario where you want to override the schedule and stop things - say for a holiday period etc - override_status sounds like it would work and if someone starts an instance manually it would just get stopped at the end time of the normal schedule. I am not seeing this behavior. It is definitely picking up the override status and state of the instance correctly as you can see from the log entries, but it does not stop the instance.

DEBUG : Desired state for instance from schedule "sydney-office-hours" is stopped, last desired state was stopped, actual state is running
DEBUG : Schedule override_status value is stopped, desired state is stopped

**Comment by lovepurohit** [comment]
Hi,

It was not working with the override_status as override_status only works the first time. I have achieved this by using enforced field. I have created a schedule with both override_status and enforced, that allows the instance to be stopped everytime it gets starts.

If anyone wants to launch it manually, they'll need to change the tag on the resource and they can start working on the resource again.
